src/components/shared/inputs/button.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import styles from './button.scss'
+
+type Props = {
+  label: string
```

Comment:
This isn't a string, it's something else 

```suggestion
  children: React.ReactNode
```

This allows you to pass in it's child like 

```javascript
<Button>My button text!</Button>
```

instead